src/app/store/networks/networks.ts

```diff
@@ -0,0 +1,5 @@
+import { atom } from 'jotai';
+
+import { defaultCurrentNetwork, Network } from './networks.slice';
+
+export const currentNetworkAtom = atom<Network>(defaultCurrentNetwork);
```

Comment:
We can derive this atom now, rather than setting duplicate state. See wallet state file where we use `get(storeAtom)`

Something like 

```ts
export const currentNetworkAtom = atom<Network>(get => {
  const store = get(storeAtom);
  // here you can possibly even use a selector to pull current network
});
```

Reply:
Good point, thx.

Reply:
We can't actually derive this atom bc we aren't saving the request network to state anywhere. This is handled in our hooks so just deriving an atom from state doesn't work. I got pretty far into it before I realized this. In the test-app, the app request account header will always load the wallet network rather than the one sent by the request. I think we can definitely remove this atom with an additional refactor I can tackle next though.

Reply:
> We can't actually derive this atom bc we aren't saving the request network to state anywhere.

I hadn't realised this, though we do actually have the params in an atom https://github.com/hirosystems/stacks-wallet-web/blob/8f1972438a9e6024300fe0c77f6a41a32e6ee06c/src/app/store/common/initial-route-search-params.ts#L3

Up to you how to handle this, though would be great to avoid duplicated "dummy" atoms, and these have proven problematic, even though intended as temporary.

Reply:
That is true, will refactor to this ...def think it is better to avoid setting a temp atom if can.